applications/serial_lte_modem/doc/NRFCLOUD_AT_commands.rst

```diff
@@ -67,23 +68,22 @@ When the ``<signify>`` parameter is not specified, it does not signify the locat
    The application executes the AT command in a best-effort way.
 
 .. note::
-   The application supports nRF Cloud cloud2device appId ``DEVICE`` to gracefully disconnect from cloud:
+   The application supports nRF Cloud cloud2device appId ``DEVICE`` to gracefully disconnect from the cloud:
 
    * cloud2device schema::
 
        {"appId":"DEVICE", "messageType":"DISCON"}.
 
-   There is no response sending to nRF Cloud for this appId.
+   There is no response sent to nRF Cloud for this appId.
 
 Unsolicited notification
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 ::
 
-   #XNRFCLOUD: <ready>,<signify>
+   #XNRFCLOUD: <ready>,<send_location>
 
-* The ``<ready>`` value indicates whether the nRF Cloud connection is ready or not.
-* The ``<signify>`` value indicates whether the location info will be signified to nRF Cloud or not.
+Refer to the read command.
```

Comment:
We can actually re-use content from the read command, so I'd suggest using it here:
```suggestion
.. include:: NRFCLOUD_AT_commands.rst
   :start-after: #XNRFCLOUD: <ready>,<send_location>,<sec_tag>,<device_id>
   :end-before: * The ``<sec_tag>`` value indicates the ``sec_tag`` used for accessing nRF Cloud.
```
This _should_ get just the two definitions needed.

Reply:
You can also add tags, though they can sometimes have problems when used inside lists. The tags should be decently specific and need to have an empty line both before and after the tag. 
```
.. tag_name_start

content to be re-used

.. tag_name_end
```

Reply:
Oh, you don't need to have the `..`, but otherwise yes. It's basically adding RST comments to the file, and just referencing the contents of those instead of a content line.